src/Components/Header/Header.js

```diff
@@ -25,6 +25,7 @@ const Header = ({ handleTextfieldChange }) => {
   const [openShare, setOpenShare] = useState(false);
   const [openEmailResults, setOpenEmailResults] = useState(false);
   const [isLanguageSelectOpen, setIsLanguageSelectOpen] = useState(false);
+  const userLanguage = navigator.language?.toLocaleLowerCase();
```

Comment:
Some users might have a region code attached to their language code. For example, if a user has a language code of `es-ar` that means that they Speak Spanish from Argentina, but we would still want to show them MFB in Spanish. Also, `.toLocaleLowerCase()` can be different depending on the users language, so it is better to use `.toLowerCase()` here in order to prevent any weird bugs. More info [here](https://stackoverflow.com/questions/34283414/difference-between-tolocalelowercase-and-tolowercase)